### DISCUSSION TITLE
Generate virtual data from real data

### DISCUSSION THREAD

**Discussion by cbldlpl** [microsoft/AirSim#3509] [Ideas]
Could airsim generate synthetic data from real data which can be comparable to real data？

I found that airsim could create fully virtual environment dataset when I read paper about TartanAir Dataset. I want to generate a dataset including real scenes and virtual scenes. And the virtual scenes are made from real scenes by changing the conditions like lighting or movement patterns.
Will it work? And if that works, what kind of data can it get?
Looking forward to your reply. Your reply is very important to me! THANKS!

**Comment by jonyMarino** [comment]
Hi @cbldlpl! This is not currently part of the AirSim project. I suppose that in that paper they would have used synthetic data together with AirSim all in Unreal Engine. But not that they used AirSim to generate it. This is a project that may interest you:
https://msr-reconstruction.azurewebsites.net/

**Reply by cbldlpl** [reply]
Can AirSim bridge the gap between virtual and real world scenarios? I mainly want to acquire the virtual scene construction of SLAM algorithm with very few real scene data.
